tests/behat/features/cu_advanced_design_bundle/blocktitleicons.feature

```diff
@@ -42,7 +38,8 @@ And I fill in "edit-conditions-plugins-path-values" with "about-apples"
 And I select "block" from "edit-reactions-selector"
 And I click the "#edit-reactions-plugins-block-selector-text-block a.fieldset-title" element
 And I check "edit-reactions-plugins-block-selector-text-block-checkboxes-bean-learn-more"
-And I click the ".context-blockform-regionlabel-sidebar_second a" element
+# And I click the ".context-blockform-regionlabel-sidebar_second a" element COULDN'T FIND APPARENTLY
```

Comment:
Sounds good, but there's no need to keep this comment in the test.

Reply:
you right; thanks for catching